pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -502,3 +510,74 @@ func checkForPodSecondaryNetworkAttachement(pod *corev1.Pod) (string, bool) {
 		return netObj, false
 	}
 }
+
+// reconcileSecondaryInterfaces restore interfacestore and cniCache when agent restart.
+func (pc *PodController) reconcileSecondaryInterfaces(pIfacestore interfacestore.InterfaceStore) error {
+	err := pc.initializeInterfaceStore()
+	if err != nil {
+		klog.ErrorS(err, "Failed to initialize the secondary bridge interface store")
+		return err
+	}
+
+	knownInterfaces := pIfacestore.GetInterfacesByType(interfacestore.ContainerInterface)
+	for _, containerConfig := range knownInterfaces {
+		event := types.PodUpdate{
+			IsAdd:        true,
+			PodName:      containerConfig.ContainerInterfaceConfig.PodName,
+			PodNamespace: containerConfig.ContainerInterfaceConfig.PodNamespace,
+			ContainerID:  containerConfig.ContainerInterfaceConfig.ContainerID,
+		}
+		pc.processCNIUpdate(event)
+	}
+
+	secondaryInterfaces := pc.interfaceStore.GetInterfacesByType(interfacestore.ContainerInterface)
+	for _, containerConfig := range secondaryInterfaces {
+		containerId := containerConfig.ContainerID
+		_, exists := pIfacestore.GetContainerInterface(containerId)
+		if !exists {
+			pc.interfaceStore.DeleteInterface(containerConfig)
```

Comment:
You should delete the OVS port and release the IP, etc. Check `removeInterfaces` (consider adding a shared removeInterface() for the common code of deleting a single interface).